### PR TITLE
Add the new Blog link to the top Navbar

### PR DESCRIPTION
**Pull request by alexventuraio**
### Desktop

image

### Mobile

Screenshot 2023-08-09 at 10 51 39

### CODE REVIEW

File: app/views/reimagine2/devpost/global_nav/mobile/_world_menu_entries.html.erb [hunk 1]
```diff
@@ -11,6 +11,10 @@
   <% end %>
 </li>
 <li class="divider"></li>
+<li>
+  <%= link_to 'Blog', 'https://info.devpost.com/blog', class: 'main-link', data: { role: 'blog' } %>
```
Comment (devpost-mzheng) [inline app/views/reimagine2/devpost/global_nav/mobile/_world_menu_entries.html.erb:15]: what is the role for? it's here but missing above

Comment (alexventuraio) [reply app/views/reimagine2/devpost/global_nav/mobile/_world_menu_entries.html.erb:15]: I just followed the convention from the other links but I think I'm going to remove it.

Comment (devpost-mzheng) [reply app/views/reimagine2/devpost/global_nav/mobile/_world_menu_entries.html.erb:15]: oh i see, i think it is for setting the active state. The blog will never the active state while in platform, so we can remove it

Comment (devpost-mzheng) [reply app/views/reimagine2/devpost/global_nav/mobile/_world_menu_entries.html.erb:15]: remove in both views

**Review by devpost-mzheng — APPROVED** [review]

**Review by jchubacca — APPROVED** [review]

**Pull request merged**